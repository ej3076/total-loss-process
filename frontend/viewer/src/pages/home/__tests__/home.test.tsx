import React from 'react';
import { create } from 'react-test-renderer';

import Home from '../';

describe('<Home />', () => {
  it('should render correctly', () => {
    const tree = create(<Home />);
    expect(tree).toMatchSnapshot();
  });
});
