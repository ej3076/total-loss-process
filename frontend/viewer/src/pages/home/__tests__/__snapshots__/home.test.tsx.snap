// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render correctly 1`] = `
<div
  className="container"
>
  <main
    className="main main"
    role="main"
  >
    <div
      className="hero"
    >
      <h1>
        Welcome to ClaimTrack
      </h1>
      <div>
        Powered by 
        <strong>
          Hyperledger Sawtooth
        </strong>
      </div>
    </div>
    <div
      className="text"
    >
      <p>
        ClaimTrack is a generic "viewer" application designed to showcase how multiple independent nodes can hook into and use data from a single blockchain network. This application enforces simple read-only access control to non-private information related to "claims" that are being tracked on the Sawtooth blockchain network.
      </p>
      <p>
        Along with the general "viewer" application, this application also provides a demo of how peers on the network may use "subscriptions" to get realtime feedback and notifications for transactions on the blockchain. This could potentially be used for sending alerts to parties involved with specific claims, or for automatically forwarding the claims' "status" when certain criteria are met.
      </p>
    </div>
  </main>
</div>
`;
